micro-rdk-installer/src/nvs/data.rs

```diff
@@ -53,7 +53,7 @@ impl ViamFlashStorageData {
                 namespace_idx,
             },
             NVSKeyValuePair {
-                key: "PASSWORD".to_string(),
+                key: "WIFI_PASSWORD".to_string(),
```

Comment:
Did the old version ever work? If so, won't this change break something? (Hopefully the answer is "the old way never worked, and the new way fixes it.")

Reply:
Before this PR there was never a micro-rdk server for the old version to work against. This was just the key my c test script for the previous PR was reading and I felt I should make the key more specific